.github/workflows/create-release-branch.yml

```diff
@@ -5,17 +5,19 @@ jobs:
     if: ${{github.ref_name == 'stage'}}
     runs-on: ubuntu-20.04
     steps:
+      - uses: actions-cool/check-user-permission@v2
+        id: check_admin
+        with:
+          require: 'admin'
+          username: ${{ github.triggering_actor }}
       - uses: actions/checkout@v3
         with:
           token: ${{ secrets.MERGE_TOKEN }}
           fetch-depth: 0
       - name: create release branch
+        if: steps.check_admin.outputs.require-result
```

Comment:
disable `release/*` branch protection before this step and enable it back after the creation